src/test/regress/expected/data_types_0.out

```diff
@@ -0,0 +1,182 @@
+CREATE SCHEMA data_types;
```

Comment:
> It seems like there is a problem with Postgres14 with SELECT DISTINCT
COUNT. The issue is reported to Postgres and an alternative output is
added. We can remove the alternative output when the issue is fixed on
PG. If this is not an issue on PG(which is unlikely) we should consider
some other solution.

Is it still a problem on PG? Can you share the link to bug report?

It feels unlikely that PG regress on such a simple query?

Reply:
I think it is, as far as I know it is not merged yet, I forwarded you the email